tests/fixtures/config.pytest.enzh.yml

```diff
@@ -0,0 +1,62 @@
+experiment:
```

Comment:
Question: We're getting more and more configs. Do you think we should file an issue to make more of the values in them optional so our configs can be more minimal for tests like this? I'm a bit concerned about maintaining a large list of configs when we want to change things.

Reply:
I don't like this either. See https://github.com/mozilla/translations/issues/597. The goal of this PR is to test CJK, we can refactor and consolidate the configs later. I imagine we could just copy the ci config and replace the target language to "zh" in tests.